resources/assets/js/websockets/index.js

```diff
@@ -0,0 +1,17 @@
+const ws = require('adonis-websocket-client')
```

Comment:
@phembarl You have to make this as a service it needs to hide its internal implementation and how it's work is can only be imported into vuex modules